From d1301df2352507f662f105fe25742dab5c65d556 Mon Sep 17 00:00:00 2001
From: Anthony Rabbito <hello@anthonyrabbito.com>
Date: Fri, 10 Sep 2021 08:13:15 -0400
Subject: [PATCH 1/2] fix: for flathub

See thread for versioning - https://invent.kde.org/education/artikulate/-/merge_requests/7?diff_id=158022#5c4cdd1800d4fd59884c2bf4df4586444cbd760b_167_168

Squashed following:

commit b053ad7d4e88c54473b667284577f88156b04841
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 23:16:58 2021 -0400

    fix: syntax

    Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>

commit c057a533d1c65949bf7b7ce14b02777bcde0fdfe
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 22:28:23 2021 -0400

    fix: add oars

    Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>

commit eb10a8896c26b9466bf96ce87cccdc2f06ee0aed
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 22:27:50 2021 -0400

    Revert "fix: appstream metadata"

    This reverts commit ee63b7381c25a55fe2f27d3cbb457e01ae813e9a.

commit ee63b7381c25a55fe2f27d3cbb457e01ae813e9a
Author: Anthony Rabbito <ted437@gmail.com>
Date:   Fri Sep 3 22:33:01 2021 -0400

    fix: appstream metadata

    * Add oars rating
    * Add release

    Signed-off-by: Anthony Rabbito <ted437@gmail.com>

Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>
---
 org.kde.artikulate.appdata.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/org.kde.artikulate.appdata.xml b/org.kde.artikulate.appdata.xml
index 6403f4c..74c7e5e 100644
--- a/org.kde.artikulate.appdata.xml
+++ b/org.kde.artikulate.appdata.xml
@@ -164,4 +164,8 @@ SPDX-License-Identifier: CC0-1.0
   <provides>
     <binary>artikulate</binary>
   </provides>
+  <releases>
+    <release version="1.0.0" date="2021-09-02"/>
+  <releases>
+  <content_rating type="oars-1.1"/>
 </component>
-- 
2.32.0


From 84d3ef1124cb4d26fb677933d67b7db34f3f7442 Mon Sep 17 00:00:00 2001
From: Anthony Rabbito <hello@anthonyrabbito.com>
Date: Fri, 10 Sep 2021 08:15:36 -0400
Subject: [PATCH 2/2] fix: syntax

Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>
---
 org.kde.artikulate.appdata.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/org.kde.artikulate.appdata.xml b/org.kde.artikulate.appdata.xml
index 74c7e5e..7bad400 100644
--- a/org.kde.artikulate.appdata.xml
+++ b/org.kde.artikulate.appdata.xml
@@ -166,6 +166,6 @@ SPDX-License-Identifier: CC0-1.0
   </provides>
   <releases>
     <release version="1.0.0" date="2021-09-02"/>
-  <releases>
+  </releases>
   <content_rating type="oars-1.1"/>
 </component>
-- 
2.32.0

